app/views/home/_find_growers.html.haml

```diff
@@ -0,0 +1,8 @@
+.find-growers.home-banner
+
+  #placesmap
```

Comment:
@Skud : We talked about making this full-width, which I think is a great idea. It's a little tricky though because the application layout nests everything in a Bootstrap `container` class, which is not designed to be full-width. Did you mention a work-around for this? I thought you did, but perhaps I hallucinated that. If not, I'm happy to move that `container` class out of the application layout and into respective templates. Alternatively, we could make a second application layout for the homepage, but that feels weird to me.